scripts/DeployFraxOFTProtocol/mfrax-aptos-move.layerzero.config.ts

```diff
@@ -58,120 +58,329 @@ function generateContractConfig(lzConfig: lzConfigType[]) {
+                            optionType: ExecutorOptionType.LZ_RECEIVE,
+                            gas: 80_000, // gas limit in wei for EndpointV2.lzReceive
+                            value: 0, // msg.value in wei for EndpointV2.lzReceive
+                        },
+                    ],
+                    sendLibrary: sendLibrary,
+                    receiveLibraryConfig: {
+                        // Required Receive Library Address on Aptos
+                        receiveLibrary: receiveLibrary,
+                        // Optional Grace Period for Switching Receive Library Address on Aptos
+                        gracePeriod: BigInt(0),
                     },
-                    ulnConfig: {
-                        // The number of block confirmations to wait on Aptos before emitting the message from the source chain.
-                        confirmations: BigInt(5),
-                        // The address of the DVNs you will pay to verify a sent message on the source chain.
-                        // The destination tx will wait until ALL `requiredDVNs` verify the message.
-                        requiredDVNs: ['0x2b696b3ee859b7eb624e1fd5de49f4d3806f49862f1177d6827fd1beffde9179', '0xdf8f0a53b20f1656f998504b81259698d126523a31bdbbae45ba1e8a3078d8da'],
-                        // The address of the DVNs you will pay to verify a sent message on the source chain.
-                        // The destination tx will wait until the configured threshold of `optionalDVNs` verify a message.
-                        optionalDVNs: [],
-                        // The number of `optionalDVNs` that need to successfully verify the message for it to be considered Verified.
-                        optionalDVNThreshold: 0,
+                    // Optional Receive Library Timeout for when the Old Receive Library Address will no longer be valid on Aptos
+                    // receiveLibraryTimeoutConfig: {
+                    //     lib: '0xbe533727aebe97132ec0a606d99e0ce137dbdf06286eb07d9e0f7154df1f3f10',
+                    //     expiry: BigInt(1000000000),
+                    // },
+                    sendConfig: {
+                        // executorConfig: {
+                        //     maxMessageSize: 10_000,
+                        //     // The configured Executor address on Aptos
+                        //     executor: '0x15a5bbf1eb7998a22c9f23810d424abe40bd59ddd8e6ab7e59529853ebed41c4',
+                        // },
+                        ulnConfig: {
+                            // The number of block confirmations to wait on Aptos before emitting the message from the source chain.
+                            confirmations: BigInt(5),
+                            // The address of the DVNs you will pay to verify a sent message on the source chain.
+                            // The destination tx will wait until ALL `requiredDVNs` verify the message.
+                            requiredDVNs: requiredDstDVNs,
```

Comment:
Do requiredDVNs need to be sorted in alphabetical order like they do in Solidity?

Reply:
The LZ tools does the sorting of DVNs in ascending order